I think most stuff is done

UnevenCoder opened the pull request:
created a new component
fixed indentation
removed unnecessary documentation [still a work in progress]
removed useless lines

UnevenCoder commented:
@BOLT04  lemme if anything more needs to be changed :)

eddiejaoude commented:
I am confused, is this PR the same as the previous one? https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/pull/74

BOLT04 reviewed (commented):
Hi @UnevenCoder 😃, I tested the other PR locally and it had a different UI. I didn't see the responsiveness but I remember you said that is a work in progress.
If you could post a screenshot of what you tested that would be great 👍?

BOLT04 commented:
> 
> 
> I am confused, is this PR the same as the previous one? #74

I think it is @eddiejaoude but with the latest changes, not sure if @UnevenCoder closed it because of the commit message or another reason.

UnevenCoder commented:
> I am confused, is this PR the same as the previous one? #74

It is the same , I deleted the branch by mistake ... and also added the changes which @BOLT04 asked

> Hi @UnevenCoder 😃, I tested the other PR locally and it had a different UI. I didn't see the responsiveness but I remember you said that is a work in progress.
> If you could post a screenshot of what you tested that would be great 👍?
![image](https://user-images.githubusercontent.com/49345531/94454314-6978c780-01ba-11eb-826f-0c071ec8e33f.png)

I haven't made it responsive just added what ever you asked 
will work on responsiveness tomorrow

UnevenCoder commented:
responsiveness fixed @BOLT04

eddiejaoude commented:
@UnevenCoder can you take screenshots of the web page with the different responsive views and add it to the PR description or as a comment please

UnevenCoder commented:
![image](https://user-images.githubusercontent.com/49345531/94527598-e437f600-023f-11eb-9b89-a49077298932.png)
![image](https://user-images.githubusercontent.com/49345531/94527680-029df180-0240-11eb-9947-396bc41bdbda.png)
done :)

eddiejaoude commented on src/App.svelte:
```diff
@@ -51,8 +51,68 @@
+      desc: 'Fullstack open source DevRel! I love code and the community :). GitHub Star ⭐ program 🤓',
+      github:'https://github.com/eddiejaoude',
+      website:'http://youtube.com/eddiejaoude?sub_confirmation=1',
+      img:"https://avatars3.githubusercontent.com/u/624760?s=460&v=4",
```
Formatting and quotes are not consistent.

Please use single quotes and have a space after the `:`

UnevenCoder replied:
doing it !

UnevenCoder replied:
done @eddiejaoude

eddiejaoude commented on src/Moderators.svelte:
```diff
@@ -0,0 +1,118 @@
+    class="grid grid-cols-1 gap-5 sm:grid-cols-1 md:grid-cols-1 lg:grid-cols-1 xl:grid-cols-2"
+  >
+
+  {#each  moderatorsData as data}
```
This seems different usage to the existing card we have. 

I think the card should be reusable and only be a single card. The `each` loop should be outside the card. 

Similar to this...

```ts
{#each cardData as cardItem}
   <Card {...cardItem} />
{/each}
```

@BOLT04 what do you think?

UnevenCoder replied:
sure that works too I skipped the work of exporting variables .. if u want to add / subtract anything you can change the data

BOLT04 replied:
I agree @eddiejaoude, we might want to use this card component for another context. So the component should only have from `<div class="bg-gray-0 h-32 md:h-48 lg:h-48">` forward. The HTML above that div goes to `App.svelte`.

I'd suggest to rename the file to something more generic, instead of `Moderators.svelte`. But it can be merged as is right now 😃.

UnevenCoder replied:
Cool is it merged yet ?

BOLT04 replied:
it is not merged yet @UnevenCoder, I was under the impression you were still making changes.
Do you agree with the requested change in these comments? We can merge this as is and refactor in another PR if you prefer.

BOLT04 commented on src/Moderators.svelte:
```diff
@@ -0,0 +1,118 @@
+      </div>
+    </div>
+  {/each}
+</div>
+</div>
```
are these two divs necessary? They are on the same level of indentation 😅 so it's a bit confusing. I think what is not indented is the content of the grid.
This is not a blocker for the PR, simply a comment to keep in mind later.

UnevenCoder replied:
WIll keep it in mnd for sure

BOLT04 reviewed (commented):
Left some comments, but looks good 👍

BOLT04 commented on src/App.svelte:
```diff
@@ -51,8 +51,68 @@
+    {
+      name : "Praveen Kumar ",
+      desc : "Microsoft MVP, Web Developer and Computer Software & UX Architect. Mentor - Full Stack @Thinkful-Ed.",
+      github : "https://github.com/praveenscience",
+      website : "https://praveen.science/",
+      img : "https://avatars1.githubusercontent.com/u/1830380?s=460&u=74697d8b1cbf3e16adec7b411369afbd53ce4864&v=4",
+      alt : "@praveenscience"
+           
+    },
+    {
+      name : "David" ,
+      desc : "Job: Software Developer Consultant I love programming, more specifically w/ Kotlin... it\'s just the best. Very interested in Web Dev and building discord bots",
+      github : "https://github.com/BOLT04",
+      website : "https://bolt04.github.io/react-ultimate-resume/",
+      img : "https://avatars1.githubusercontent.com/u/18630253?s=460&u=36cdb692666236ff5db7aa3c71e58327bdbdecc7&v=4",
+      alt : "@BOLT04"
+    },
+    {
+      name : "Eddie Jaoude" ,
+      desc : "Fullstack open source DevRel! I love code and the community :). GitHub Star ⭐ program 🤓",
+      github : "https://github.com/eddiejaoude",
+      website : "http://youtube.com/eddiejaoude?sub_confirmation=1",
+      img : "https://avatars3.githubusercontent.com/u/624760?s=460&v=4",
+      alt : "@eddiejaoude"
+    },
+    {
+      name : "Petra",
+      desc : "Linux and blockchain. Looking for ways to change the world.",
+      github : "https://github.com/metaverde",
+      website : "https://metaverde.github.io/",
+      img : "https://avatars1.githubusercontent.com/u/32101027?s=460&u=a3204811c6965f28996b6bcd36bdc800216d5dfe&v=4",
+      alt : "@metaverde"
+    },
+    {
+      name : "Amandine Flachs",
+      desc : "",
+      github : "https://github.com/AmandineFlachs",
+      website : "https://www.flachsconsulting.com/",
+      img : "https://avatars0.githubusercontent.com/u/29413522?s=460&v=4",
+      alt : "@AmandineFlachs"
```
I think what @eddiejaoude mean in this [comment](https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/pull/76#discussion_r496483006) was to have the same format used in `cardsData`:
```suggestion
      name: "Stephen Mount",
      desc: "After 10+ years in software engineering, I am coming up with ideas and things to reverse engineer things for the new wave of ARM etc etc",
      github: "https://github.com/stemount",
      website: "https://ste.london/",
      img: "https://avatars1.githubusercontent.com/u/150512",
      alt: "@stemount"
    },
    {
      name: "Michael Mba",
      desc: "Linux Systems Integrator | Open Source Enthusiast | Photographer | Gamer || Starting my Python journey.",
      github: "https://github.com/mikeysan",
      website: "https://mikeysan.hashnode.dev/",
      img: "https://avatars0.githubusercontent.com/u/13338176?s=460&u=d66a31344752bb3fa882a59b3effb10e084f226c&v=4",
      alt: "@mikeysan"
    },
    {
      name: "Praveen Kumar ",
      desc: "Microsoft MVP, Web Developer and Computer Software & UX Architect. Mentor - Full Stack @Thinkful-Ed.",
      github: "https://github.com/praveenscience",
      website: "https://praveen.science/",
      img: "https://avatars1.githubusercontent.com/u/1830380?s=460&u=74697d8b1cbf3e16adec7b411369afbd53ce4864&v=4",
      alt: "@praveenscience"
    },
    {
      name: "David" ,
      desc: "Job: Software Developer Consultant I love programming, more specifically w/ Kotlin... it\'s just the best. Very interested in Web Dev and building discord bots",
      github: "https://github.com/BOLT04",
      website: "https://bolt04.github.io/react-ultimate-resume/",
      img: "https://avatars1.githubusercontent.com/u/18630253?s=460&u=36cdb692666236ff5db7aa3c71e58327bdbdecc7&v=4",
      alt: "@BOLT04"
    },
    {
      name: "Eddie Jaoude" ,
      desc: "Fullstack open source DevRel! I love code and the community :). GitHub Star ⭐ program 🤓",
      github: "https://github.com/eddiejaoude",
      website: "http://youtube.com/eddiejaoude?sub_confirmation=1",
      img: "https://avatars3.githubusercontent.com/u/624760?s=460&v=4",
      alt: "@eddiejaoude"
    },
    {
      name: "Petra",
      desc: "Linux and blockchain. Looking for ways to change the world.",
      github: "https://github.com/metaverde",
      website: "https://metaverde.github.io/",
      img: "https://avatars1.githubusercontent.com/u/32101027?s=460&u=a3204811c6965f28996b6bcd36bdc800216d5dfe&v=4",
      alt: "@metaverde"
    },
    {
      name: "Amandine Flachs",
      desc: "",
      github: "https://github.com/AmandineFlachs",
      website: "https://www.flachsconsulting.com/",
      img: "https://avatars0.githubusercontent.com/u/29413522?s=460&v=4",
      alt: "@AmandineFlachs"
```

BOLT04 commented on src/App.svelte:
```diff
@@ -1,6 +1,6 @@
 <script>
   import Card from "./Card.svelte";
-
+  import ModeratorCard from './Moderators.svelte'
```
```suggestion
  import ModeratorCard from './Moderators.svelte';
```

BOLT04 commented on src/Moderators.svelte:
```diff
@@ -0,0 +1,118 @@
+<script>
+  export let moderatorsData
```
```suggestion
  export let moderatorsData;
```

BOLT04 reviewed (dismissed):
I made some suggestions with the missing points. The rest can come in another PR as improvements. The goal of the next PR is to refactor `Moderators.svelte` to a more re-usable component, and check if those two divs are necessary.

@eddiejaoude can you give your approval before we merge? Just want to make sure everything is fine 😃.

eddiejaoude commented:
I trust you @BOLT04 🤓

eddiejaoude commented:
Merged 👍 

- I think next step would be to create an issue for having 1 data item sent to the component rather than the array
